### PR TITLE
Refactor `highlight-collaborators-and-own-conversations` to use `selector-observer`

### PR DESCRIPTION
**Pull request by yakov116**
#2671 

Test on

https://github.com/sindresorhus/refined-github/pulls

### CODE REVIEW

File: source/features/highlight-collaborators-and-own-conversations.tsx [hunk 1]
```diff
@@ -53,6 +50,10 @@ void features.add({
 	include: [
 		pageDetect.isRepoConversationList
 	],
+	exclude: [
+		() => select.exists('.blankslate')
```
Comment (fregante) [inline source/features/highlight-collaborators-and-own-conversations.tsx:54]: We have this on github-url-detection

Comment (yakov116) [reply source/features/highlight-collaborators-and-own-conversations.tsx:54]: Only isEmptyRepoRoot

Comment (fregante) [reply source/features/highlight-collaborators-and-own-conversations.tsx:54]: We should probably drop that one and just use this function everywhere

**Review by fregante — APPROVED** [review]

File: source/features/highlight-collaborators-and-own-conversations.tsx (hunk 1 above)
Comment (yakov116) [reply source/features/highlight-collaborators-and-own-conversations.tsx:54]: https://github.com/fregante/github-url-detection/commit/0c36c4c08636174cc039b077680ff90acb3b5d37

Comment (yakov116) [reply source/features/highlight-collaborators-and-own-conversations.tsx:55]: @fregante how to proceed?

Comment (fregante) [reply source/features/highlight-collaborators-and-own-conversations.tsx:55]: `npm deprecate *`

**Review by fregante — CHANGES_REQUESTED** [review]
👎  This doesn't seem to have benefits, according to https://github.com/sindresorhus/refined-github/issues/3084#issuecomment-683345700 — that is, this is not a "ajaxed hot-area"

Thoughts?

**Comment by yakov116** [comment]
There is one benefit here that the request is sent before dom ready and then applied as soon as dom ready. I could do await dom ready, that would accomplish the same thing.

**Comment by fregante** [comment]
Indeed, but that's unrelated to `selector-observer`. You can send a PR just for that

**Comment by yakov116** [comment]
In other words you mean don't wait for dom ready and then after the request do await dom loaded?

**Comment by fregante** [comment]
Correct

**Pull request closed without merging**